relationships/synthesis/MOBILE-VIDEO-to-MEDIA-STREAMING.yml

```diff
@@ -0,0 +1,41 @@
+relationships:
+  - name: mobileAppContainsMediaStreamingApp
+    version: "1"
+    origins:
+      - Mobile Monitoring
+    conditions:
+      - attribute: parentAppId
+        present: true
+      - attribute: eventType
+        startsWith: Video
+        caseSensitive: true
+      - attribute: src
+        anyOf: ["Android", "IOS"]
+    relationship:
+      expires: P75M
+      relationshipType: CONTAINS
+      source:
+        buildGuid:
```

Comment:
you don't need to use `buildGuid` on both source and target, if the entity guid comes in the telemetry you can just extract the guid from there, like [this](https://github.com/newrelic/entity-definitions/blob/main/relationships/synthesis/APM-APPLICATION-to-DATABASE.yml#L14-L15)